Remotion/Core/Extensions/Logging/Log4Net/Log4NetLogger.cs

```diff
@@ -15,27 +15,118 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 //
 using System;
+using System.Globalization;
 using log4net.Core;
-using Microsoft.Extensions.Logging;
+using log4net.Util;
+using MicrosoftLogging = Microsoft.Extensions.Logging;
+using RemotionLogging = Remotion.Logging;
```

Comment:
I thought we would do type alising, not namespace alisaing 
MicrosoftLogger
RemotionLogger
MicorosftLogLevel
RemotionLogLevel
etc